### PR TITLE
fix setenv usages in documentation examples

### PR DESCRIPTION
**Pull request by AndreyNautilus**
## Thanks for contributing a pull request!

If you are contributing for the first time or provide a trivial fix don't worry too
much about the checklist - we will help you get started.

## Contribution checklist:

(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)

- [ ] wrote descriptive pull request text
- [ ] added/updated test(s)
- [ ] updated/extended the documentation
- [ ] added relevant [issue keyword](https://help.github.com/articles/closing-issues-using-keywords/)
      in message body
- [ ] added news fragment in [changelog folder](../tree/master/docs/changelog)
  * fragment name: `<issue number>.<type>.rst` for example (588.bugfix.rst)
  * `<type>` is must be one of `bugfix`, `feature`, `deprecation`,`breaking`, `doc`, `misc`
  * if PR has no issue: consider creating one first or change it to the PR number after creating the PR
  * "sign" fragment with "by :user:`<your username>`"
  * please use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files - by :user:`superuser`."
  * also see [examples](../tree/master/docs/changelog)
- [ ] added yourself to `CONTRIBUTORS` (preserving alphabetical order)

### CODE REVIEW

File: docs/example/basic.rst [hunk 1]
```diff
@@ -192,21 +192,25 @@ multiple PyPI servers, using ``PIP_EXTRA_INDEX_URL`` environment variable:
...
         mypackage
 
 This configuration will install ``docutils`` from the default
-Python PYPI server and will install the ``mypackage`` from
+Python PyPI server and will install the ``mypackage`` from
 our index server at ``https://mypypiserver.org`` URL.
 
+.. warning::
+
+  Using an extra PyPI index as a fallback for installing private packages may cause security issues.
```
Comment (jugmac00) [inline docs/example/basic.rst:210]:
```suggestion
  Using an extra PyPI index for installing private packages may cause security issues.
```

I would remove the `as a fallback` as the intention would be to use the private server as the main server for private packages, which then causes the security issues when somebody registers the package on the public PyPI server.

File: docs/example/basic.rst [hunk 2]
```diff
@@ -192,21 +192,25 @@ multiple PyPI servers, using ``PIP_EXTRA_INDEX_URL`` environment variable:
...
+  For example, if ``mypackage`` is registered in the default PyPI index, PIP will install ``mypackage``
```
Comment (jugmac00) [inline docs/example/basic.rst:211]:
```suggestion
  For example, if ``mypackage`` is registered with the default PyPI index, pip will install ``mypackage``
```

**Review by gaborbernat — APPROVED** [review]

File: docs/example/basic.rst [hunk 3]
```diff
@@ -192,21 +192,25 @@ multiple PyPI servers, using ``PIP_EXTRA_INDEX_URL`` environment variable:
...
+  from default PyPI index, not from custom one.
```
Comment (jugmac00) [inline docs/example/basic.rst:212]:
```suggestion
  from the default PyPI index, not from the custom one.
```

File: docs/changelog/1999.doc.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Examples in documentation use ``setenv`` correctly (in ``[testenv]`` sections, not in ``[tox]``).
```
Comment (jugmac00) [inline docs/changelog/1999.doc.rst:1]:
```suggestion
Update examples in the documentation to use ``setenv`` in the ``[testenv]`` sections, not wrongly in the ``[tox]`` main section.
```

**Review by jugmac00 — CHANGES_REQUESTED** [review]
Looks good to me, with a couple of minor suggestions.

**Review by jugmac00 — APPROVED** [review]

**Comment by jugmac00** [comment]
Thanks a lot!

**Pull request merged**